pyproject.toml

```diff
@@ -292,87 +300,19 @@ optional = true
 version = ">=0.39, <0.43"
 optional = true
 
 [tool.poetry.dependencies.pymongo]
 version = ">=3.8,<3.11"
-extras = ["tls", "srv"]
+extras = [ "tls", "srv",]
 
 [tool.poetry.dev-dependencies.pytest-sanic]
 git = "https://github.com/RasaHQ/pytest-sanic"
 branch = "fix_signal_issue"
 
-[tool.mypy]
-# some libraries do not have type hints; ideally we remove
-# this at some point but it will require a lot of efforts from us and library owners
-mypy_path = "./stubs"
-ignore_missing_imports = true
-show_error_codes = true
-warn_redundant_casts = true
-warn_unused_ignores = true
-disallow_untyped_calls = true
-disallow_incomplete_defs = true
-disallow_untyped_defs = true
-disallow_untyped_decorators = true
-# FIXME: should be switch to true (default) after fixing implicit Optional in the codebase
-no_implicit_optional = false
-# FIXME: working our way towards removing these
-# see https://github.com/RasaHQ/rasa/pull/6470
-# the list below is sorted by the number of errors for each error code, in decreasing order
-disable_error_code = ["arg-type"]
-
-[tool.pytest.ini_options]
-# Function starting with the following pattern are considered for test cases.
-python_functions = "test_"
-filterwarnings = ["ignore::ResourceWarning:ruamel[.*]"]
-log_cli_level = "WARNING"
-log_cli = true
-# asyncio_mode = "auto" FIXME: https://rasahq.atlassian.net/browse/ATO-859
-markers = [
-    "skip_on_windows",              # mark a test as a test that shouldn't be executed on Windows.
-    "skip_on_ci",                   # mark a test to skip in the CI
-    "sequential",                   # mark tests than cannot be ran in parallel (e.g. because they need access to the same resource).
-    "category_cli",                 # Marks tests for the command-line interface.
-    "category_core_featurizers",    # Marks tests for components which featurize trackers for the policies.
-    "category_policies",            # Marks tests for the dialogue policies.
-    "category_nlu_featurizers",     # Marks tests for NLU components which add features to the user message.
-    "category_nlu_predictors",      # Marks tests for NLU components which make predictions based on the user message (e.g. intent classifiers or entity extractors).
-    "category_full_model_training", # Marks tests which test the training workflow for entire models (instead of single policies / components).
-    "category_other_unit_tests",    # Default marker for unit tests which have no `category_` marker applied to them.
-    "category_performance",         # Marker for performance tests (e.g. speed or detecting memory leaks).
-    "flaky",                        # mark a test as flaky (i.e., fails non-deterministically).
-]
-# see https://pypi.org/project/pytest-timeout/
-timeout = 60
-timeout_func_only = true
-
-[tool.bandit]
-skips = [
-    "B104", # checks for binding 0.0.0.0 interface, which should be fine for containers.
-    "B301", # checks for pickle usage, which is a necessary evil.
-]
-
-[tool.ruff]
```

Comment:
Just curious about how these formatting changes to `pyproject.toml` came about 🤔 I think the previous content is preserved, still creates the false impression that there's a considerable diff.

Reply:
I had to do a `poetry lock --no-update` after updating the sdk version otherwise tensorflow was getting updated to  2.11.1 in the lockfile. Can that be the reason?

Reply:
Did you update the `sdk` with `poetry add`? That could explain it, that's the only command I know that changes `pyproject.toml`.

Reply:
no, just manually updated the value in `pyproject.toml`